components/common/feeds/AuthorFeed.tsx

```diff
@@ -18,10 +18,12 @@ export const useAuthorFeedQuery = ({
 }) => {
   const { data: repoData } = useRepoAndProfile({ id })
   const labelerAgent = useLabelerAgent()
+  const appviewAgent = useAppviewAgent()
 
   return useInfiniteQuery({
     queryKey: ['authorFeed', { id, query, typeFilter }],
     queryFn: async ({ pageParam }) => {
+      let isFromAppview = false
```

Comment:
From the looks of it (and for someone that is not familiar with how everything works here), the loop body could therorically return results from both the labeler AND the appview. Because that is the case, using a single `boolean` to store `isFromAppview` could be misleading.
Additionnaly, it seems that if the labeller errors on the first loop iteration, and more items are still required, the labeller will un-necessarily be tryed again during the following loop iterations.

Reply:
the only time when the loop may use both labelerAgent and appviewAgent is if someone blocks the labeler WHILE the loop is being executed which is super unlikely so I think we can get away with this for a bit. other approach may be to check for a block record first and then use the right agent which is an additional api req for every author feed load so I think the current approach is a tiny bit better though, kinda complex.

good catch on the extra failing call, cleaned it up.